rules/dropbox_rules/dropbox_external_share.yml

```diff
@@ -0,0 +1,152 @@
+AnalysisType: rule
+Description: Dropbox item shared externally
+DisplayName: "Dropbox External Share"
+Enabled: true
+Filename: dropbox_external_share.py
+Severity: Medium
+Tests:
+    - ExpectedResult: false
+      Log:
```

Comment:
this test case can use[ this mocking example from the same gsuite yam](https://github.com/panther-labs/panther-analysis/blob/master/rules/gsuite_reports_rules/gsuite_drive_visibility_change.yml#L168-L173)l file to show the impact of a user setting ALLOWED_DOMAINS